forgit.plugin.zsh

```diff
@@ -184,6 +184,11 @@ forgit::rebase() {
 
 forgit::checkout::branch() {
     forgit::inside_work_tree || return 1
+    [[ $# -ne 0 ]] && {
+        cmd="git checkout -b $*"
+        eval "$cmd"
+        return
```

Comment:
`eval` seems not necessary here? And `cmd` should be declared as local otherwise it will be exported to user's environment.

Can we can simplified these into one line?
```bash
[[ $# -ne 0 ]] && git checkout -b "$*" && return
```